hack/update_codegen.sh

```diff
@@ -10,15 +10,13 @@ set -o pipefail
 export GO111MODULE=on
 version_and_repo=$(go list -f '{{if .Replace}}{{.Replace.Version}} {{.Replace.Path}}{{else}}{{.Version}} {{.Path}}{{end}}' -m k8s.io/code-generator)
 VERSION="$(echo "${version_and_repo}" | cut -d' ' -f 1 | rev | cut -d"-" -f1 | rev)"
-CODEGEN_REPO="$(echo "${version_and_repo}" | cut -d' ' -f 2)"
 REPO_ROOT="${REPO_ROOT:-$(git rev-parse --show-toplevel)}"
 CODE_GEN_DIR="${REPO_ROOT}/hack/code-gen/$VERSION"
 
 if [[ -d ${CODE_GEN_DIR} ]]; then
     echo "Using cached code generator version: $VERSION"
 else
-    echo "Cloning code-generator from ${CODEGEN_REPO}"
```

Comment:
Why get rid of this? Does it cease to work with the `replace` removed?

Reply:
Cloning didn't work for me, because the URL provided in `go.mod` isn't the one of the repo that needs to be used. That's why reverted this back to using a hardcoded repo. Because the first line of `git clone` is "Cloning $repo", I also removed this `echo`.

Reply:
Hm, right. I tried to see how we could make it work in both cases but I got lost. `go help modules` says:

> Module code layout
> For now, see https://research.swtch.com/vgo-module for information
> about how source code in version control systems is mapped to
> module file trees.

And that doc is about the prototype and does not show an easy way to transform the module `Path` into a clonable URL. So let's stick with what you have for now.